### DISCUSSION TITLE
How do I add additional files to 'Find Config File'?

### DISCUSSION THREAD

**Discussion by jcarpenter12** [LazyVim/LazyVim#1981] [Q&A] [answered]
Hi all,

I recently moved over to lazy vim and I absolutely love it. Please bear with me as I am not very familiar with how to manage some of the lua configurations but I have been able to configure most of what I want so far. 

My question is, Lazy Vim by default provides a way to jump to the lua configuration files  using `<leader>fc` to quickly pull them up and modify them. I really like this feature but is there any way to add additional files/directories to this shortcut? I would like for example to be able to update my '.zshrc' file from this view, without having to search for it through the fuzzy finder.

Any help greatly appreciated,

Thanks

**Comment by dpetka2001** [comment, marked as answer]
This keymap just opens `~/.config/nvim/` directory with Telescope and you are able to search the files within that directory. I'm not aware if Telescope is able to open directly a file. So your options would be:
1) Open Telescope in the directory where your `.zshrc` is located to search for it
2) Create a mapping to open directly the file, something like `vim.keymap.set("n", "<leader>fC", "<cmd>e ~/.zshrc<cr>")`

**Reply by jcarpenter12** [reply]
Thank you for your answer, that makes sense. I will give it a go
